### DISCUSSION TITLE
/versions slash command no longer opens if no record found

### DISCUSSION THREAD

**Discussion by Kolymorph** [arnoudkooi/SN-Utils#273] [Ideas]
Hi,

I've noticed that using /versions no longer opens new window if the record doesn't have any recorded versions - is this configurable or will this be the default setup going forward? The command was pretty useful to get to the queried version table fast and easily editable in case of records, which have different version name syntax (like choice lists)

**Comment by arnoudkooi** [comment]
You clear the _Overwrite tatget URL_ field.
This should bring back the old behavior

<img width="802" alt="image" src="https://user-images.githubusercontent.com/840781/180952832-f39d21a2-1798-4ba8-827c-91a3d4b19b6a.png">

**Reply by Kolymorph** [reply]
Ah great, thanks a lot
